lib/AuroraLip/Texture/Formats/FIPAFTEX.cs

```diff
@@ -0,0 +1,56 @@
+﻿using AuroraLip.Archives.Formats;
+using AuroraLip.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using static AuroraLip.Texture.Formats.TXE;
+using static AuroraLip.Texture.J3D.JUtility;
+
+
+namespace AuroraLip.Texture.Formats
+{
+    // The actual file type is likely 'FIPA'
+    // with a switch on subtypes (ex: 'FTEX')
+    // but that would be more complicated
+    // to deal with!
+    // Since these can contain one or more
+    // TPLs, we derive from that class
+    // to reuse that code
+    internal class FIPAFTEX : TPL
+    {
+        new public bool CanRead => true;
+
+        new public bool CanWrite => false;
+
+        new public string Magic => magic;
+
+        private const string magic = "FIPAFTEX";
+
+        new public static bool Matcher(Stream stream, in string extension = "")
+        {
+            if (!stream.MatchString(magic))
+                return false;
+
+            return true;
+        }
+
+        new public bool IsMatch(Stream stream, in string extension = "")
+            => Matcher(stream, extension);
```

Comment:
'''
public class FIPAFTEX : TPL , IMagicIdentify
{

        public override bool CanWrite => false;

        public override string Magic => magic;

        private const string magic = "FIPAFTEX";

        public override bool IsMatch(Stream stream, in string extension = "")
            => Matcher(stream, extension);
'''

Reply:
Are you getting rid of the `Matcher` here?  I'm not sure what you are trying to show